reporting-app/app/controllers/demo/certifications_controller.rb

```diff
@@ -47,4 +57,30 @@ def form_params
     params.require(:demo_certifications_create_form)
           .permit(Demo::Certifications::CreateForm.attribute_names.map(&:to_sym))
   end
+
+  def create_ex_parte_activities
```

Comment:
is there a reason we are not using the same creation service for the demo as we are for the main API endpoint, this looks like duplicate... this could get outta sync